### ISSUE TITLE
Should we remove the stable branch?

### ISSUE THREAD

**Issue by Bertram25** [OpenDungeons/OpenDungeons#695]
As raised by @akien-mga , the stable branch and the latest 0.x branches have got the same content.
Should we then remove the stable branch? Or is there a need for it we didn't see?

Feel free to comment. :)

**Comment by akien-mga** [comment]
tl;dr: I'd say drop _stable_ and create _0.6_ as soon as we want to start stabilizing things for the next milestone.

I think the initial idea with the stable branch was that _development_ would be a messy place full of untested commits with regressions, _stable_ would be the ready-to-release WIP version of our next milestone, while the _0.x_ branches would hold bug fixes and eventually allow us to push bugfix releases (e.g. 0.5.1).

As it turned out, our workflow based on using exclusively PRs and making sure that they don't introduce regressions (or work in priority on fixing the regressions as soon as we spot them), the _development_ branch is already near to what we might have envisioned for _stable_.

The situation in which we might want to start using this _stable_ branch might be if we decided to merge some heavy refactoring in development, that would be meant for the next milestone but would still be a WIP. But IMO, in such case we might just decide branch development to create the corresponding _0.x_ branch before merging the WIP content.

**Comment by hwoarangmy** [comment]
I agree, stable branch can be dropped. If someone wants to download latest stable, he would have already seen the latest version on our website anyway and he would easily find it looking at the branch names

**Comment by Bertram25** [comment]
Fine, it will be done shortly. :)

**Comment by Bertram25** [comment]
It is done.

**Comment by oyvindln** [comment]
The stable branch is basically a relic from the sourceforge days.
